### PR TITLE
feat/account

### LINKED ISSUE #2: [4.1.1.1] Admin/Normal User (Accounts) - C: Add new user account

### LINKED ISSUE #7: [4.1.5.1] Admin (Accounts) - R: Query all user accounts
Image

### LINKED ISSUE #8: [4.1.5.2] Admin (Accounts) - D: Delete user accounts

### LINKED ISSUE #9: [4.1.5.3] Admin (Accounts) - U: Modify user accounts
- Approve: Role of user
- ~Delete: Add column to store deleted status~

### PR DESCRIPTION
**Pull request by isabellepakyishan**
Admin/User - Add new account: Closes #2 
Admin - Account Management: Closes #7 
Admin - Update account information + Approve account + Modify account password: Closes #9 
Admin - Delete account: Closes #8

### CODE REVIEW

File: accounts-management.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8" />
+    <title>Home | TSA0 Records Systems</title>
```
Comment (jiachenyee) [inline accounts-management.html]: Aren't we changing the title line?

File: change-password.html [hunk 2]
```diff
@@ -0,0 +1,139 @@
+<!doctype html>
+<html lang="en">
+
+    <head>
+
+        <meta charset="utf-8" />
+        <title>Home | TSAO Capstone Records System</title>
```
Comment (jiachenyee) [inline change-password.html]: Also page title

**Review by jiachenyee — CHANGES_REQUESTED** [review]
Are we changing the page titles?

File: accounts-management.html (hunk 1 above)
Comment (Kuan-Zher) [inline accounts-management.html]: Title should be "TSA Office Records Systems"

File: change-password.html (hunk 2 above)
Comment (Kuan-Zher) [inline change-password.html]: Title should be "TSA Office Records Systems"

**Review by Kuan-Zher — CHANGES_REQUESTED** [review]
Change the titles to the correct title

**Review by jiachenyee — APPROVED** [review]
thanks for the change! looks good!

**Review by Kuan-Zher — APPROVED** [review]
Lgtm

**Pull request merged**